### DISCUSSION TITLE
Custom blocked services

### DISCUSSION THREAD

**Discussion by binarweb** [AdguardTeam/AdGuardHome#4729] [Ideas]
While the custom filters works great, having to disable a service requires commenting several lines of custom filters.

It would be nice to "bundle" together a bunch of DNS rules and give them a name and have them listed in "Blocked services".

This option would allow fast disable/enable of certain custom blocked services.

If we would go further, custom blocked services could also be created using block lists.

**Comment by fernvenue** [comment]
Duplicate of https://github.com/AdguardTeam/AdGuardHome/issues/1692.
